x-pack/plugins/observability/public/pages/alerts/alerts_flyout/alerts_flyout.test.tsx

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
This is incomplete, I only added as much as I needed to test my changes. Expanding on this should be easy :)